src/server/stream_family.cc

```diff
@@ -287,17 +287,31 @@ OpResult<vector<GroupInfo>> OpListGroups(const DbContext& db_cntx, string_view k
   return result;
 }
 
+constexpr int8_t create_group_opts_mkstream = 1 << 0;
```

Comment:
Please change to `kCreateOptMkstream`

Reply:
Done! Also changed the type from `int8_t` to `uint8_t` here..